### PR TITLE
Create Mpumalanga COVID 19 Expenditure Report–Health Donation

### PR DESCRIPTION
**Pull request by tristynferreiro**
#5

### CODE REVIEW

File: Mpumalanga COVID 19 Expenditure Report–Health Donation [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+Hand sanitiser Spray,19-Jun,Goods and services,AngloAmerican,Middelburg Pharmaceutical Dep,t                1 000.00
+Hand Paper towels,24-Apr,Goods and services,Delmas Development Centre(NPO,Bernice Samuel Hospital,-
+HP Laser jet Multifunction 137FW,01-Jul,rPor i7n2t2e0r,South32 SA Coal Holdings Propriet,rNyk Laimngiatelad District,4 232.54
+Industrial Paper Towels,15-May,Goods and services,Glencore Operations S.A (Pty) Ltd,Nkangala District,37 500.00
+Internet router bundle for 06 mon,01-Jul,Goods and services,South32 SA Coal Holdings Propriet,rNyk Laimngiatelad District,5 085.30
+Sanitisers,(blank),Goods and services,STEAM POWER S&E AFRICA,Nkangala District,79 500.00
+KN95/ FFP2 Masks,10-Jun,Goods and services,FORD,Nkangala District,-
+KN95/ FFP2 Masks,09-Jul,Goods and services,Naspers,Nkangala District,-
+7 x Logitech Wireless USB Combo,01-Jul,aCrodm &p Mutoeur sDee-1sk2t8op,South32 SA Coal Holdings Propriet,rNyk Laimngiatelad District,3 566.78
+7 x Macaroni Versitas Lightweight,01-Jul,Gposoed sB acnkdp saecrkvices,South32 SA Coal Holdings Propriet,rNyk Laimngiatelad District,1 522.78
```
Comment (georgezee) [inline Mpumalanga COVID 19 Expenditure Report–Health Donation]: Spot checks showed some issues with the above 2 rows:
`aCrodm &p Mutoeur sDee-1sk2t8op` should be `Computer Desktop`
`Gposoed sB acnkdp saecrkvices` should be `Goods and Services`

**Comment by georgezee** [comment]
There are some issues with how some of the text came through in the CSV, e.g.:
 
text-issues-pr-235

**Comment by tristynferreiro** [comment]
Should be fixed now! Sorry about that

**Comment by tristynferreiro** [comment]
@jbothma should I remove the first row and the grand totals from this too?

**Comment by georgezee** [comment]
Thanks for the update @candidcreature. I've removed the additional header row.

There seem to be some other issues with the file as well. There are two grand total rows, which yes, should be removed. Looking closer, the two grand totals are for the same amount (12 399 140.09). If I understand this correctly, some pages are a "payment register", whilst others are a "donation register", so I'm not sure how they should be split up.

Also, the data starts at page 30 of the PDF with "300 x bath towels",  and the data from the previous page on the PDF is missing (page 29 - starting with "100 x Liquid hand soap".

**Comment by tristynferreiro** [comment]
@georgezee Hi there, 

The Donation and Payment registers have been split into two different pull requests and files. 

Sorry about missing that table. I will have a look at it and fix it. I also noticed that I did all the other tables in one file but they probably should have been split into the different sectors i.e. Education, Tourism, eg.(This was the first file I did so I have learnt a lot since then, sorry)
 I will do all of that shortly. Just focusing on getting National data and the Free State done first.

**Comment by georgezee** [comment]
No worries @candidcreature - I think this file is a tricky one as the columns aren't as clearly delineated as they should be.
Thanks for the update.

**Pull request closed without merging**